index.html

```diff
@@ -49,6 +49,7 @@
   font-family: monospace;
   font-size: large;
   list-style-type: none;
+  overflow-x: scroll;
```

Comment:
Wouldn't `auto` be better? https://developer.mozilla.org/en-US/docs/Web/CSS/overflow

Reply:
Sounds good , still serves the purpose with elegance